repose-aggregator/components/filters/valkyrie-authorization/src/test/scala/org/openrepose/filters/valkyrieauthorization/ValkyrieAuthorizationFilterTest.scala

```diff
@@ -43,6 +43,7 @@ import org.scalatest.mock.MockitoSugar
 import org.scalatest.{BeforeAndAfter, FunSpec}
 
 import scala.collection.JavaConversions._
+import scala.collection.JavaConverters._
```

Comment:
Why are we using both **implicit** (`JavaConversions`/older) and **explicit** (`JavaConverters`/more capable) Java/Scala conversions?
Do we want to standardize on one or the other?